### DISCUSSION TITLE
How to capture desktop app traffic

### DISCUSSION THREAD

**Discussion by wimpykid719** [mitmproxy/mitmproxy#5104] [Q&A]
Hi, I'm new kind of this capture app.

I installed mitmproxy using brew and did setup.
They worked and capturing chrome traffic when I use chrome.
but I want to capture any other desktop app like docker, dropbox, mac system access to API.

I saw charles proxy could capture any other desktop app.
How to set up in mitmproxy?

**Comment by Prinzhorn** [comment]
You probably want to set up transparent mode, see https://docs.mitmproxy.org/stable/concepts-modes/

**Reply by wimpykid719** [reply]
Thank you for suggestion.
I tried this command `mitmproxy --listen-port 8888 --mode transparent` and changed proxy settings of mac to web proxy（HTTP） `0.0.0.0:8888` and web proxy（HTTPS）to `0.0.0.0:8888` but noting display to my mitmproxy. 
and I can't connect to network with chrome and any app.

**Reply by wimpykid719** [reply]
I tried this too. `mitmproxy --mode transparent`
and nothing change my network proxy settings.
because of this.

> Make sure you have not explicitly configured an HTTP proxy on the client. This is not needed in transparent mode.

[Troubleshooting Transparent Mode](https://docs.mitmproxy.org/stable/concepts-modes/#troubleshooting-transparent-mode)

**Reply by Prinzhorn** [reply]
> and nothing change my network proxy settings.

Did you remove the mac proxy settings or no?

**Reply by wimpykid719** [reply]
<img width="677" alt="http・https-proxy-settings" src="https://user-images.githubusercontent.com/23703281/155835400-9067b02e-f939-42b8-bbad-43e79d33f750.png">

 I just checked off this settings.

**Reply by wimpykid719** [reply]
I think I need to follow this settings.
but I don't know these command well. so I'm worried how to fix the settings if I ended up to use mitproxy transparent mode.
[https://docs.mitmproxy.org/stable/howto-transparent/#macos](https://docs.mitmproxy.org/stable/howto-transparent/#macos)
